lpcompiler.pas

```diff
@@ -1762,6 +1780,23 @@ function TLapeCompiler.ParseMethod(FuncForwards: TLapeFuncForwards; FuncHeader:
       if (Result.Statements = nil) or (not (Result.Statements.Statements[Result.Statements.Statements.Count - 1] is TLapeTree_StatementList)) then
         Expect(tk_kw_Begin, False, False);
 
+      if (lcoHints in FOptions) then
```

Comment:
Doing this here might result in missing some implicitly used variables, such as `Result` in an `Exit(123)` expression.